_includes/tutorials/dynamic-output-topic/kstreams/markup/dev/run-consumer.adoc

```diff
@@ -0,0 +1,20 @@
+////
+  This is a sample content file for how to include a console consumer to the tutorial, probably a good idea so the end user can watch the results
+  of the tutorial.  Change the text as needed.
+
+////
+
+Now that you have produced some orders, you should set up a consumer to view the results. But in this case, you need to start two consumers as the Kafka Streams
```

Comment:
```suggestion
Now that you have produced some orders, you should set up a consumer to view the results. In this case, you need to start two consumers as the Kafka Streams
```

Looks like you can also join this line with the line below. Not sure if asciidoc renders an unwanted line break here or not, as-is.